src/ds323x/alarms.rs

```diff
@@ -2,6 +2,7 @@
 
 use super::super::{BitFlags, Ds323x, Error, Hours, Register};
 use super::{decimal_to_packed_bcd, hours_to_register};
+use crate::ds323x::{NaiveTime, Timelike};
 use interface::{ReadData, WriteData};
 
 /// Parameters for setting Alarm1 on a day of the month
```

Comment:
Could you document here how the values of invalid inputs are adapted when these are ignored by the selected matching  strategy?